Purger/DefaultPurger.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+namespace SofaScore\Purgatory\Purger;
+
+
+class DefaultPurger implements PurgerInterface
+{
+
+    public function purge(iterable $urls): void
+    {
+        throw new \Exception('Not implemented');
```

Comment:
@zlatkoverk You should rather throw a `BadMethodCallException` because those types of exceptions aren't meant to be handled (they extend `LogicException`).

As a plus, the IDE will stop complaining about unhandled exceptions. 😄